web/src/components/countryhistorytitle.jsx

```diff
@@ -0,0 +1,26 @@
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { useTranslation } from '../helpers/translation';
+import React from 'react';
+import { formatTimeRange } from '../helpers/formatting';
+
+const Text = styled.span`
+  font-size: 1.1em;
+`;
+
+export const CountryHistoryTitle = ({ translationKey }) => {
+  const { __, i18n } = useTranslation();
+  const aggregation = useSelector((state) => state.application.selectedTimeAggregate);
+  const localExists = i18n.exists(`${translationKey}.${aggregation}`, { fallbackLng: i18n.language });
```

Comment:
Oh yes, this may be a bit odd. 

Right now our fallback language is EN, this means that localExist will be true in most cases, as it will just go to the english language and see that it's there. Our desired functionality is that if there is no translation for e.g "DE" then it should say false here. Setting the fallbackLng to the same language is one way of ensuring it doesn't go to the english one.

Do you know of a better method to check if there is a specific translation for a specific language?

Reply:
Hmm, I think [getFixedT](https://www.i18next.com/overview/api#getfixedt) could work for this, but it doesn't really make things simpler, so it's probably better to just stick with your solution here :)